src/_macosx.m

```diff
@@ -47,6 +47,7 @@
 #ifndef COMPILING_FOR_10_5
 static int ngc = 0;    /* The number of graphics contexts in use */
 
+#include <Carbon/Carbon.h>
```

Comment:
Carbon is the old api. Cocoa is the new one https://en.wikipedia.org/wiki/Carbon_(API) so I don't think so